src/Microsoft.OpenApi.Hidi/Program.cs

```diff
@@ -127,5 +128,29 @@ static async Task Main(string[] args)
             //// Wait for logger to write messages to the console before exiting
             await Task.Delay(10);
         }
+
+        /// <summary>
+        /// This event is raised when the user presses either of the two breaking key combinations: Ctrl+C or Ctrl+Break keys.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="eventArgs"></param>
+        private static void Console_CancelKeyPressEvent(object sender, ConsoleCancelEventArgs eventArgs)
```

Comment:
we should also create a cancellation token source, call cancel on the cts, and pass the cancellation token to command.InvokeAsync

Reply:
I've found a cleaner way to implement this using the invocation context by getting an `IConsole` using `InvocationContext.Console`, getting a CancellationToken (triggering Ctrl+C [cancellation handling](https://docs.microsoft.com/en-us/dotnet/standard/commandline/handle-termination)) by calling InvocationContext.GetCancellationToken() - [Reference](https://github.com/dotnet/command-line-api/issues/1750#issuecomment-1146502468)